parsl/monitoring/db_manager.py

```diff
@@ -544,7 +544,10 @@ def start(self,
                 if reprocessable_last_resource_messages:
                     self._insert(table=STATUS, messages=reprocessable_last_resource_messages)
             except Exception:
-                logger.exception("Exception in db loop: this might have been a malformed message, or some other error. monitoring data may have been lost")
+                logger.exception(
+                    "Exception in db loop: this might have been a malformed message,"
```

Comment:
here's something I didn't notice on my last review! there's a common mistake with multiline strings (that I often make too) -- there is no space between the command at the end of this line and the word "or" on the next line!
(compare to how you split up format_string in interchange.py with spaces at the end of each string.